buildSrc/subprojects/plugins/src/main/kotlin/org/gradle/gradlebuild/unittestandcompile/UnitTestAndCompilePlugin.kt

```diff
@@ -108,14 +107,16 @@ class UnitTestAndCompilePlugin : Plugin<Project> {
 
     private
     fun Project.addGeneratedResources(gradlebuildJava: UnitTestAndCompileExtension) {
-        val classpathManifest by tasks.creating(ClasspathManifest::class)
+        val classpathManifest = tasks.createLater("classpathManifest", ClasspathManifest::class.java) {
```

Comment:
We should add a `createLater()` method that does not take a configure action, rather than leave this empty action here (but as a later PR).

Reply:
I would encourage introducing a local `creatingLater` delegate to preserve the API and keep the definitions refactorable.

Reply:
Yes, that's exactly what I was thinking Adam